pkg/aggregator/aggregator.go

```diff
@@ -897,10 +908,41 @@ func (agg *BufferedAggregator) run() {
 				}
 			}
 			tlmFlush.Add(1, event.eventType, state)
+		case event := <-agg.contLcycleIn:
+			agg.contlcycleDequeueOnce.Do(func() { go agg.dequeueContainerLifecycleEvents() })
```

Comment:
Any particular reason to start it only on-the-fly? It could make sense to start it ni the Demultiplexer everytime if `UseContainerLifecycleForwarder == true`, WDYT?

Reply:
Agreed 👍

Reply:
I'll move it to `demux.Run()` unless you think of a better place

Reply:
Perfect, don't forget to have it in the `demux.Stop()` as well if needed.

Reply:
thanks for confirming! no need for `demux.Stop()` if we agree on closing the current stopChan :D